ml_model/model/results/compare_results.py

```diff
@@ -0,0 +1,172 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+from matplotlib.lines import Line2D
+
+import csv
+import json
+
+from scipy.stats import wilcoxon
+from scipy.stats import normaltest
+
+ROOT_PATH = "/Users/nadeeshan/Documents/Spring2023/ML-Experiments/complexity-verification-project/ml_model/model/"
+
+df = pd.read_csv(ROOT_PATH + 'results/raw_results.csv')
+
+use_SMOTE = False
+
+if use_SMOTE:
+    SMOTE_TAG='SMOTE'
+else:
+    SMOTE_TAG='NO_SMOTE'
+
+############################
+## FILTER OVERALL RESULTS ##
+############################
+## filter overall results
+overall_df = df.query("iteration == 'overall' and use_smote == " + str(use_SMOTE))
+## remove duplicate configs ##
+overall_df = overall_df.drop_duplicates(subset=['target','tp_c','tn_c','fp_c','fn_c','tp_cw','tn_cw','fp_cw','fn_cw','precision_c','recall_c','f1_c','precision_cw','recall_cw','f1_cw'], keep='first')
```

Comment:
This is probably not needed after we fix the problem with having two sets of best hyperparameters